server/etcdserver/api/etcdhttp/health.go

```diff
@@ -15,10 +15,13 @@
 package etcdhttp
 
 import (
+	"bytes"
 	"context"
 	"encoding/json"
 	"fmt"
 	"net/http"
+	"path"
+	"strings"
```

Comment:
Would you please rebase the main branch? we fix the linter issue.

At the Line 29, the `"go.etcd.io/etcd/api/v3/etcdserverpb"` is duplicated with `pb "go.etcd.io/etcd/api/v3/etcdserverpb"`. Not sure that the linter will be broken after merged. 
I think it's safe to rebase and rerun the linter. Thanks